Makefile-tests.am

```diff
@@ -287,6 +293,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
+installed_test_programs = $(_installed_or_uninstalled_test_programs)
+check-local:
+	echo "NOTE: Installed tests are enabled; to run them, make install, then: gnome-desktop-testing-runner -p 0 libostree/"
```

Comment:
Depends on your value of "normal developer testing" :smile:.  I use [atomicws](https://pagure.io/atomic-ws) where installing devel tools on the host is an anti-pattern; or to say it positively, I do build/tests in containers, and hence I `sudo make install` for ostree and other tools, and therefore installed tests work there too.

Anyways, I can add `--enable-installed-tests=exclusive` or so. 

I addressed your point `2)` in the commit message.  Some tests we have are *only* uninstalled and I don't have a sane way to run *just* those without introducing new infrastructure.